src/typescript/frontend/src/lib/store/name-store.ts

```diff
@@ -118,54 +113,31 @@ export const initializeEventStore = (): NameState => {
 
 export const defaultState: NameState = initializeEventStore();
 
-type Set = (
-  nextStateOrUpdater: NameStore | Partial<NameStore> | ((state: WritableDraft<NameStore>) => void),
-  shouldReplace?: boolean | undefined
-) => void;
-type Get = () => NameStore;
-
-const getNameMap = (set: Set, get: Get, addresses: string[]): ANSMap => {
-  const map = get().names;
-  const uncachedNames: string[] = [];
-  for (const address of addresses) {
-    if (!map.has(address)) {
-      uncachedNames.push(address);
-    }
-  }
-  if (uncachedNames.length > 0) {
-    for (const address of uncachedNames) {
-      if (!ansPromiseMap.has(address)) {
-        const x = fetchANSName(address)
-          .then((res) => ({
-            name: res,
-            expiry: new Date().getTime() + ANS_CACHE_TIME,
-          }))
-          .then((res) => {
-            set((state) => {
-              state.names.set(address, res);
-              ansPromiseMap.delete(address);
-              if (ansPromiseMap.size === 0) {
-                saveANSMap(state.names);
-              }
-            });
-          });
-        ansPromiseMap.set(address, x);
-      }
-    }
-  }
-  return get().names;
-};
-
 export const createNameStore = (initialState: NameState = defaultState) => {
   return createStore<NameStore>()(
     immer((set, get) => ({
       ...initialState,
-      getMapWithNames: (addresses) => {
-        return getNameMap(set, get, addresses);
-      },
-      getResolverWithNames: (addresses) => {
-        const map = getNameMap(set, get, addresses);
-        return (address) => map.get(address)?.name ?? address;
```

Comment:
For future reference @CRBl69, this was the offending line. Feel free to resolve once you see this